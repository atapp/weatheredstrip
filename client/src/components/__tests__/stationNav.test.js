import React from 'react';
import { render, cleanup } from 'react-testing-library'
import ReactDOM from 'react-dom';
import StationNav from '../stationNav.react';
import testData from './test_data.json'
import 'jest-dom/extend-expect';
afterEach(cleanup)

it('stationNav renders properly', () => {
  const { container } = render(<StationNav data={ testData }/>)
  const stationNav = container.firstChild
  expect(stationNav).toMatchSnapshot()
});

// Need function testing for selection of other stations.
