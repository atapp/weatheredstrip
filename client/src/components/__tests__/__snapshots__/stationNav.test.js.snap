// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`stationNav renders properly 1`] = `
<div
  id="navbar"
>
  <div
    class="navbar-content"
  >
    <div
      class="station"
    >
      CYMX
    </div>
    <div
      class="station"
    >
      EGPK
    </div>
  </div>
</div>
`;

exports[`stationNav renders properly 2`] = `
<div
  id="navbar"
/>
`;
